### ISSUE TITLE
New Tool Ideas

### ISSUE THREAD

**Issue by tderkert** [TypeResources/typeresources.github.io#10]
- [ ] **Pangram viewer:** View pangrams in selected fonts. Possibly identify characters that aren't included in the font's glyphset. [reddit thread](https://www.reddit.com/r/typography/comments/4r9h7p/pangram_resource/)
- [ ] **Glyph Support Viewer:** Lists and display all fonts glyphs in grid. Show language support etc.

**Comment by larsenwork** [comment]
So these would be "standalone" tools similar to how glyph gazer is, no? If we start reusing codeparts then we could just extract those as a node_module to be included i subprojects

**Comment by tderkert** [comment]
Yeah, standalone is how though of it. How does the node_modules work?

**Comment by larsenwork** [comment]
We could pull out reusable code in separate repo with a package.json and just `npm install link-to-repo`

**Comment by tderkert** [comment]
Ok, sounds good. Shouldn't we do the same from the whole Glyphs Gazer tool as well? Will be easier with issue tracking when we have more tool than one, maybe.

**Comment by larsenwork** [comment]
I think it's easier just keeping the reusable parts as node modules. If we put the entire Glyph Gazer a node module then we would have to have a build script in Typeresources that copies the correct files from the non-synced node modules folder into the root folder - but could maybe make sense at some point

**Comment by tderkert** [comment]
Fair, that sounds more complicated than it needs to be right now.

**Comment by tderkert** [comment]
I checked Pangram Viewer as there is one already in Impallari's testing tools.

**Comment by larsenwork** [comment]
Just because it exists elsewhere doesn't mean we shouldn't do it (and maybe do it better) :neckbeard:

**Comment by etunni** [comment]
I agree :)

**Comment by tderkert** [comment]
Yeah, you're both right! Consider it unchecked again.
